src/workerd/server/server.c++

```diff
@@ -1445,6 +1446,122 @@ void Server::InspectorServiceIsolateRegistrar::registerIsolate(
 }
 
 // =======================================================================================
+namespace {
+class RequestObserverWithTracer final: public RequestObserver, public WorkerInterface {
+public:
+  RequestObserverWithTracer(kj::Maybe<kj::Own<WorkerTracer>> tracer): tracer(kj::mv(tracer)) {}
+  ~RequestObserverWithTracer() noexcept(false) {
+    KJ_IF_SOME(t, tracer) {
+      if (fetchStatus != 0) {
+        t->setFetchResponseInfo(Trace::FetchResponseInfo(fetchStatus));
+      }
+      t->setCPUTime(0 * kj::MILLISECONDS);
```

Comment:
Well why would you have to manually set the cpu and wall time of the tracer to zero?

Reply:
If we decide to not set those in workerd for now I think a comment saying why we decided that is legit, otherwise if it's a TODO that's fine as well.
But I think it'll be easier for future readers to understand if we have a clear comment here on why this is set to 0 instead of some real value.
On the other hand if you had just not set it at all then people would just assume it's not yet implemented which is fine as well. I just think setting to 0 specifically sends a message that this has been intentional with that should come an explanation.

Reply:
I'm setting them here right now to rule out the possibility of potentially uninitialized fields causing the crashes that I'm seeing in the windows CI. If I end up keeping these I will add a comment but in general none of the mechanisms for tracking cpu/wall time are implemented for the workerd version of this, at least for now, these should be zero.